src/settings/SettingsModal.tsx

```diff
@@ -61,7 +61,7 @@ export function SettingsModal({ open, onCloseClick }: { open: boolean; onCloseCl
   }
 
   return (
-    <div className="fixed left-0 top-0 flex h-screen w-screen items-center justify-center">
+    <div className="z-99 fixed left-0 top-0 flex h-screen w-screen items-center justify-center">
```

Comment:
my PR has `z-[99999]` :)

Reply:
I didn't mean to add this to this PR, I had changed the code to fix the interface while I was testing it, but then accidentally added it to my commit. I will revert it, to avoid conflicts